nfs-ganesha/build/build_rpm

```diff
@@ -117,12 +118,12 @@ if [ "$THROWAWAY" = false ] ; then
 }
 EOF
     # post the json to repo-extra json to chacra
-    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}/repos/${chacra_endpoint}/extra/
+    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}repos/${chacra_repo_endpoint}/extra/
```

Comment:
@andrewschoen one thing I noticed was that when the equivalent curl command is run in ceph-dev-build, what I have as `${chacra_url}/repo` is `${chacra_url}repo` (https://github.com/ceph/ceph-build/blob/master/ceph-dev-build/build/build_deb#L107) is this a typo/does it make a difference?